README.md

```diff
@@ -3,15 +3,24 @@
 This lambda is intended to trigger a project build on Codeship.
 It makes use of the *codeship-go* API client.
 
+** Make sure to give permissions to the Codeship user to trigger builds
+on the targeted projects
+
 ## Required Environment Variables (for the lambda config)
 
 * CS_ORGANIZATION // the Codeship organization
 * CS_PASSWORD // the Codeship password
 * CS_USERNAME // the Codeship username for authentication
-* CS_PROJECT_UUID // the uuid of the target Codeship project
-* CS_BUILD_REFERENCE // the desired reference for the build, e.g. "20.04" or "develop"
+
+
+* CS1_PROJECT_UUID // the uuid of the first target Codeship project
+* CS1_BUILD_REFERENCE // the desired reference for the build, e.g. "20.04" or "develop"
 (note: "heads/" will be prepended to this value, if it is not already there)
```

Comment:
Need to duplicate this note below, or move it lower so it applies to both `CS1_BUILD_REFERENCE` and `CS2_BUILD_REFERENCE`?